evm-types.md

```diff
@@ -665,21 +665,31 @@ Accounts
 Storage/Memory Lookup
 ---------------------
 
-`#lookup*` looks up a key in a map and returns 0 if the key doesn't exist, otherwise returning its value.
+- `#lookup` looks up a key in the contract storage and returns 0 if the key doesn't exist, otherwise returning its value.
+
+- `#lookupMemory` looks up a key in the local memory and returns 0 if the key doesn't exist, otherwise returning its value.
+
+- `#write` stores a value for a certain key in the contract storage.
```

Comment:
```suggestion
-   `#lookup` looks up a key in the contract storage and returns 0 if the key doesn't exist, otherwise returning its value.
-   `#lookupMemory` looks up a key in the local memory and returns 0 if the key doesn't exist, otherwise returning its value.
-   `#write` stores a value for a certain key in the contract storage.
```